src/App.js

```diff
@@ -1,19 +1,106 @@
 import React from 'react';
 import './App.css';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList';
+import { AddForm } from './components/AddForm';
 
 import users from './api/users';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+const preparedTodos = [...todos].map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state = {
+    usersTodos: preparedTodos,
+    chooseUser: '',
+    titleError: false,
+    userError: false,
+    title: '',
+    userId: 2,
+  };
+
+  handleChenge = (event) => {
+    const { name, value } = event.target;
+    const { title, chooseUser } = this.state;
+
+    if (title === '') {
+      this.setState({ titleError: false });
+    }
+
+    if (chooseUser === '') {
+      this.setState({ userError: false });
+    }
+
+    this.setState({ [name]: value });
+  }
+
+  addTodos = () => {
```

Comment:
```suggestion
  addTodo = () => {
```
ты добавляешь одну туду за раз